streams/src/main/java/org/apache/kafka/streams/processor/WallclockTimestampExtractor.java

```diff
@@ -15,10 +15,9 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.streams.processor.internals;
+package org.apache.kafka.streams.processor;
 
 import org.apache.kafka.clients.consumer.ConsumerRecord;
-import org.apache.kafka.streams.processor.TimestampExtractor;
 
 public class WallclockTimestampExtractor implements TimestampExtractor {
```

Comment:
While we're making this timestamp change, we should also add a docstring to `WallclockTimestampExtractor`, explaining that, if my understanding is correct, it yields processing-time semantics (= time when records are being consumed/processed).